@xen-orchestra/web-core/lib/components/CardNumbers.vue

```diff
@@ -0,0 +1,86 @@
+<!-- v1.0 -->
+<template>
+  <div class="card-numbers" :class="size">
+    <span class="label typo" :class="labelFontClass">{{ label }}</span>
+    <div class="values">
+      <span v-if="size === 'small' && max" class="value typo c2-semi-bold">{{ valueAsPercentage }}%</span>
+      <div class="value typo" :class="valueFontClass">
+        {{ value }}<span class="unit typo" :class="unitFontClass">{{ unit }}</span>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { computed } from 'vue'
+
+interface CardNumbersProps {
+  label: string
+  value: number
+  unit?: string
+  size: 'small' | 'medium'
+  max?: number
+}
+
+const props = defineProps<CardNumbersProps>()
+
+const labelFontClass = computed(() => (props.size === 'medium' ? 'c3-semi-bold' : 'c2-semi-bold'))
+
+const valueFontClass = computed(() => (props.size === 'medium' ? 'h3-semi-bold' : 'c2-semi-bold'))
+
+const unitFontClass = computed(() => (props.size === 'medium' ? 'p2-medium' : 'c2-semi-bold'))
+
+const valueAsPercentage = computed(() => {
+  if (props.max === undefined) {
+    return
+  }
+
+  return ((props.value / props.max) * 100).toFixed(1)
+})
+</script>
+
+<style lang="postcss" scoped>
+/* COLOR VARIANTS */
+.card-numbers {
+  &.small {
+    --label-color: var(--color-grey-100);
+  }
+
+  &.medium {
+    --label-color: var(--color-grey-300);
+  }
+}
+
+/* IMPLEMENTATION */
+.card-numbers {
+  display: flex;
+  gap: 0.8rem;
+  justify-content: space-between;
```

Comment:
As discussed during the call with Clémence, it was decided that values should not be separated from their labels.

Some XO 6 mock-ups have been updated but not the component in the DS. I added a comment on Figma.

```suggestion
```